samples/highcharts/demo/column-stacked/demo.js

```diff
@@ -27,9 +31,9 @@ Highcharts.chart('container', {
     },
     legend: {
         align: 'right',
-        x: -30,
+        x: -400,
```

Comment:
When the chart is small (like in visual test), the legend comes off the chart area. Better align left and with a smaller x offset.